src/test/java/FavLIst.java

```diff
@@ -22,19 +25,25 @@ public void addingFavoriteSong() {
         loginPageIntern.login();
         // first go to all songs
         favoritesList.allSongMenu();
-        //favoritesList.deleteAllFavSongs();
-        //addSongtoFavorites();
-        //String selectedSongTitle = getSongText();
-        //checkFavoriteList();
-        //String  favSong = checkSongInFavList();
-
-        //Assert.assertEquals(favSong,selectedSongTitle);
-
-        //click on the song to add to Favoritesfa
-        //get the text of the songs
-        //add song to favorites
-        //go to favorites
-        //check if the song is there
+        // add song name
+//        List<String> listSongToAdd=new ArrayList<String>();
+//
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.